ui/src/search.ts

```diff
@@ -47,7 +47,7 @@ const VALID_URN_MATCH = /^rad:git:[1-9A-HJ-NP-Za-km-z]{37}/;
 const urnConstraints = {
   format: {
     pattern: VALID_URN_MATCH,
-    message: `Not a valid project URN`,
+    message: `Not a valid project URN, perhaps you're using an old URN?`,
```

Comment:
I'm not sure this is an improvement, as-is. Users will have no idea what that means.

Reply:
> Users will have no idea what that means.

Not until they read the post ;P

How could this be improved? A link to the post (possibly to a specific URN section)?

Reply:
Hmm, I wonder if it's worthwhile to have this (to then have to eventually remove it). I think just leaving it as it was might be fine -- we can explain in the post that old URNs aren't compatible, at least they will have the context there.

Reply:
Works for me. We can also add a section to the troubleshooting docs